File: Tanmay Jaiswal/day-3/3b.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if (x<0) return false;
        long long int y = 0, z = x;

        while (x) {
            y = y*10 + x%10;
            x /= 10;
        }

        return z==y;
    }
};

File: Vinayak Sharma/day-3/3B.java
class Solution {
    public boolean isPalindrome(int x) {
      String s = String.valueOf(x);  

        int a = 0;                   
        int b = s.length() - 1; 
       
          while(a <= b)      
        {
            if(s.charAt(a) != s.charAt(b))  
                return false;
            a++;                                
            b--;                                
        }  
        
        return true;
        
    }
}
